templates/footer.hbs

```diff
@@ -1,7 +1,37 @@
 <footer class="footer">
+  {{#if settings.show_contact_us_section}}
+      <div class="contact-block">
+        <div class="footer-inner">
+          <h1>{{settings.contact_us_header}}</h1>
+          {{#link 'new_request' class='submit-a-request btn' role='button'}}
+            Contact Us
+          {{else}}
+          <span class='link-not-found'>
+            Contact Us link is not available for you.
+          </span>
+          {{/link}}
+        </div>
+      </div>
+  {{/if}}
   <div class="footer-inner">
-    {{#link 'help_center'}}{{help_center.name}}{{/link}}
-
+    <div class="company-info">
+        <img src="{{settings.mit_footer_logo}}">
```

Comment:
We should take the user to https://openlearning.mit.edu/ upon clicking this logo. This is already happening on https://mitxonline.zendesk.com/hc/en-us